nat-lab/tests/test_mesh_firewall.py

```diff
@@ -449,67 +448,30 @@ async def test_mesh_firewall_file_share_port(
             with pytest.raises(asyncio.TimeoutError):
                 await ping(connection_beta, CLIENT_ALPHA_IP, 15)
 
-        output_notifier = OutputNotifier()
-        listening_start_event = asyncio.Event()
-        sender_start_event = asyncio.Event()
-        connected_event = asyncio.Event()
+        async with NetCatServer(
+            connection_alpha, PORT, udp=True, ipv6=CLIENT_PROTO == IPProto.IPv6
```

Comment:
I think source_IP got lost somewhere for `NetCatServer()` processes.

Reply:
added `bind_ip`